ra/ra-api-next.rng

```diff
@@ -73,6 +79,11 @@
 	</choice>
 </define>
 
+<define name="description">
+	<attribute name="lang" />
+	<ref name="anyElement" />
+</define>
```

Comment:
I'm also not sure why the parameter descriptions have the anyElement reference. I'd expect a simple "text" value. I can't imagine our tooling being able to use anything else.

Reply:
I agree but this is just for backward compatibility. I would be happy to remove that in 2.x

Reply:
Sounds good